src/merkle/store/mod.rs

```diff
@@ -478,26 +478,23 @@ impl MerkleStore {
         Ok(RootPath { root, path })
     }
 
+    /// Merge two elements and add their result into the store.
```

Comment:
minor nit: We usually use 3rd person for such descriptions - i.e., "Merges two elements and adds the resulting node into the store."

Reply:
fixed